src/util/report.py

```diff
@@ -286,6 +286,18 @@ def ref(xid):
     except Exception:
         _logger.warning("Cannot announce message", exc_info=True)
 
+    # Chat window with the report will be open post-upgrade for the admin user
+    if version_between("9.0", "saas~18.2"):
+        channel_member_model = (
+            "discuss.channel.member"
+            if version_gte("saas~16.3")
+            else "mail.channel.member"
+            if version_gte("16.0")
+            else "mail.channel.partner"
+        )
+        domain = [("partner_id", "=", user.partner_id.id), ("channel_id", "=", recipient.id)]
+        registry[channel_member_model].search(domain)[:1].with_context(ctx).fold_state = "open"
```

Comment:
Isn't this going to fail if the search returns empty?

Reply:
Should be better now